testing/integration/beat_receivers_test.go

```diff
@@ -220,58 +223,52 @@ func TestAgentMonitoring(t *testing.T) {
 					return docs.Hits.Total.Value > 0
 				},
 				2*time.Minute, 5*time.Second,
-				"No documents found for type: %s, dataset: %s, namespace: %s", tc.dsType, tc.dsDataset, tc.dsNamespace)
-		}
-	})
-
-	t.Run("compare logs ingested by agent monitoring vs otel monitoring", func(t *testing.T) {
-		// skipping this because the log-path should be handled differently in windows
-		if runtime.GOOS == "windows" {
-			t.Skip("skipping this test on windows for now")
+				"agent monitoring classic no documents found for timestamp: %s, type: %s, dataset: %s, namespace: %s, query: %v", timestamp, tc.dsType, tc.dsDataset, tc.dsNamespace, tc.query)
 		}
 
-		// Not proceed with this test if monitoring logs from elastic-agent does not exist
-		monitoringLogIndex := "logs-elastic_agent-" + info.Namespace
-		require.NotPanics(
-			t, func() {
-				_ = agentDocs[monitoringLogIndex].Hits.Hits[0].Source
-			}, "monitoring logs from elastic-agent should exist before proceeding",
-		)
-
-		ctx, cancel := testcontext.WithDeadline(t, context.Background(), time.Now().Add(5*time.Minute))
-		t.Cleanup(cancel)
-
-		// beats processes and beats receivers should use a different namespace to ensure each test looks only at the
-		// right data
-		actualNamespace := fmt.Sprintf("%s-%s", info.Namespace, "otel")
-		policy.Agent.Monitoring["namespace"] = actualNamespace
-
-		// switch monitoring to the otel runtime
-		policy.Agent.Monitoring["_runtime_experimental"] = "otel"
-
-		updatedPolicyBytes, err := yaml.Marshal(policy)
-		require.NoErrorf(t, err, "error marshalling policy, struct was %v", policy)
-		t.Cleanup(func() {
-			if t.Failed() {
-				t.Logf("policy was %s", string(updatedPolicyBytes))
-			}
-		})
-
-		// 3. Install without enrolling in fleet
-		fixture, err := define.NewFixtureFromLocalBuild(t, define.Version())
+		// 3. Uninstall
+		combinedOutput, err := classicFixture.Uninstall(ctx, &atesting.UninstallOpts{Force: true})
+		require.NoErrorf(t, err, "error uninstalling classic agent monitoring, err: %s, combined output: %s", err, string(combinedOutput))
+
+		// 4. Start elastic agent monitoring in otel mode
+		receiverMonitoringTemplate := `
+outputs:
+  default:
+    type: elasticsearch
+    hosts: {{.ESEndpoint}}
+    api_key: {{.ESApiKey }}
+    preset: balanced
+
+agent.monitoring:
+  enabled: true
+  logs: true
+  metrics: true
+  use_output: default
+  namespace: {{ .Namespace }}
+  _runtime_experimental: otel
+`
```

Comment:
This looks like the exact same config as for beats processes, with the exception of the `_runtime_experimental` field. Can we just use one config with that field templated?